src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
+    get coords() {
```

Comment:
Add blank lines between methods, and properties in the object, It'll be easier to read your code. Do this in all places
```suggestion
    y: 0,
    
    get coords() {
```